### PR TITLE
Add note on app.config name changes.

### PR DESCRIPTION
**Pull request by pyrrho**

### CODE REVIEW

**Comment by macintux** [comment]
:+1:

**Comment by jhmullen** [comment]
+1

File: RELEASE-NOTES.md [hunk 1]
```diff
@@ -75,6 +75,48 @@ CS still support the older style configurations through `app.config` and
...
+Stanchion, some parameters have changed names and must be updated**.
+
+In particular, for the Riak CS `app.config`:
+- `cs_ip` and `cs_port` have been combined into `listener`.
+- `riak_ip` and `riak_pb_port` have been combined into `riak_host`.
+- `stanchion_ip` and `stanchion_port` have been combined into `stanchion_host`.
+- `admin_ip` and `admin_port` have been combined into `admin_listener`.
+- `webmachine_log_handler` has become `webmachine_access_log_handler`.
+- `{max_open_files, 50}` has been deprecated and should be replaced with
+  `{total_leveldb_mem_percent, 30}`.
```
Comment (kuenishi) [inline RELEASE-NOTES.md]: `max_open_files` is for Riak. This item should be removed.

**Comment by kuenishi** [comment]
Except single pick above, +1 from me. This is very good catch, thanks guys.

**Pull request merged**